vividus/src/test/java/org/vividus/runner/BddStepPrinterTests.java

```diff
@@ -97,9 +97,9 @@ public void testPrintToFile() throws Exception
         StepCandidate stepCandidate2 = mockStepCandidate("When", "I do '$action'", "deprecatedMethod");
         StepCandidate stepCandidate3 = mockStepCandidate("Then", "I perform '$verification'", (Method) null);
         when(candidateSteps.listCandidates()).thenReturn(List.of(stepCandidate1, stepCandidate2, stepCandidate3));
-        return List.of("                        Given initial state is '$status'",
-                             "COMPOSITE IN STEPS FILE Then I perform '$verification'",
-                             "DEPRECATED              When I do '$action'");
+        return List.of("                         Given initial state is '$status'",
```

Comment:
Could you please cover the logic related to the location?

Reply:
I'm doubtful it's really worth: module names are collected only for steps from jar-s, so I need to create a sample jar with steps, publish it somewhere and use it in unit tests, since this functionality is not crucial, I would leave it as is for now